apps/aecore/src/aec_tx_pool.erl

```diff
@@ -151,15 +155,19 @@ push(Tx) ->
     push(Tx, tx_created).
 
 -spec push(aetx_sign:signed_tx(), event()) -> ok | {error, atom()}.
-push(Tx, Event = tx_received) ->
+push(Tx, Event) ->
+    push(Tx, Event, ?DEFAULT_PUSH_TIMEOUT).
```

Comment:
I see [OTP default is 5000](http://erlang.org/documentation/doc-9.3/lib/stdlib-3.4.4/doc/html/gen_server.html#call-2) so this does not change semantics. Ok.